cmd/sensors/src/lib.rs

```diff
@@ -120,7 +123,8 @@ fn list(
                 };
 
                 println!(
-                    "{:2} {:7} {:2} {:2} {:3} 0x{:02x} {:13} {:<1}",
+                    "{:3} 0x{:03x} {:7} {:2} {:2} {:3} 0x{:02x} {:13} {:<1}",
```

Comment:
Use `#` and the formatter will automatically add `0x`
```suggestion
                    "{:3} {:#05x} {:7} {:2} {:2} {:3} {:#04x} {:13} {:<1}",
```